src/tokens/6DecimalBridgedToken.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.18;
+
+import {BridgedToken as BT} from "./BridgedToken.sol";
+/**
+ * @title BridgedToken
+ * @dev Inherits from BridgedToken but overrides decimals to be 6. To be used on tokens with 6 decimals (e.g USDC).
+ */
+
+contract BridgedToken is BT {
```

Comment:
Let's not have two contracts with the same name. Let's just modify `BridgedToken` so the decimal is immutable which can be set to any number 🙏